01.general/02.how-to-create-your-own-ad-filters/docs.en.md

```diff
@@ -1211,6 +1212,61 @@ Use `@@` to negate `$removeheader`:
 
 > **Compatibility with different versions of AdGuard.** Rules with `$removeparam` modifier are supported by AdGuard for Windows, Mac, Android, and AdGuard Browser extension for Chrome, Firefox, Edge.
 
+
+<a id="jsonprune-modifier"></a>
+#### **`$jsonprune`**
+
+`$jsonprune` rules modify the JSON response of a matching request by removing JSON items that match a modified (see below)
```

Comment:
We need a disclaimer that explains that this is only available with CoreLibs 1.10 (i.e. dev builds)